variables.tf

```
@@ -202,6 +202,12 @@ variable "diagnostic_setting_enabled_log_categories" {
   ]
 }
 
+variable "disable_advanced_threat_protection" {
+  description = "Should advanced threat protection be turned off? This setting should be enabled in subscriptions where 'New Defender for Storage' is enabled"
+  type        = bool
+  default     = false
+}
```

Follow Terraform baseline variable naming convention:

```suggestion
variable "advanced_threat_protection_enabled" {
  description = "Should Defender for Storage (classic) advanced threat protection be enabled for this Storage account?"
  type        = bool
  default     = true
}
```

Also updated description to be more similar to other variables in this module.
Feel free to suggest further changes.